compiler/one-cmds/tests/one-import-bcq_neg_003.test

```diff
@@ -21,7 +21,7 @@ filename="${filename_ext%.*}"
 
 trap_err_onexit()
 {
-  if grep -q "input model not found" "${filename}.log"; then
```

Comment:
This error message followed https://github.com/Samsung/ONE/issues/4155#issuecomment-690903095
@lemmaa , do we need to keep this or can we update this to follow new tools?

Reply:
Had a off-line talk with @lemma and it is OK to change these :)